server.ts

```diff
@@ -0,0 +1,35 @@
+///<reference path='.\Declarations\node\node.d.ts' />
```

Comment:
I was told at some point to either add the license per-file or add a license.txt to the repository root. I'll double check though.